### PR TITLE
Documentation - Improve pax/passenger intercom clarity

### PR DESCRIPTION
**Pull request by Vdauphin**
**When merged this pull request will:**
- This PR switch from "passenger" to "pax" as it is used in game.

May be just an advise could be enough? like this:
_Here we will use passenger as pax for clarity._

### CODE REVIEW

File: docs/wiki/user/vehicle-intercom.md [hunk 1]
```diff
@@ -6,9 +6,9 @@ title: Vehicle Intercom
 
 ## Vehicle Intercom
 
-Vehicle intercom consists of two separate networks depending on the vehicle type: crew and passenger intercom. Intercom gives the ability to speak to other players within the same network using a headset. This allows them to communicate without the disturbance of the vehicle's noise.
+Vehicle intercom consists of two separate networks depending on the vehicle type: crew and pax intercom. Intercom gives the ability to speak to other players within the same network using a headset. This allows them to communicate without the disturbance of the vehicle's noise.
```
Comment (TheMagnetar) [inline docs/wiki/user/vehicle-intercom.md]:
```suggestion
Vehicle intercom consists of two separate networks depending on the vehicle type: crew and passenger (pax) intercom. Intercom gives the ability to speak to other players within the same network using a headset. This allows them to communicate without the disturbance of the vehicle's noise.
```

File: docs/wiki/user/vehicle-intercom.md [hunk 2]
```diff
@@ -6,9 +6,9 @@ title: Vehicle Intercom
...
 
-While *crew intercom* is limited to crew members and players in cargo positions cannot have access to it, *passenger intercom* can be accessed by both crew and passengers. This allows the crew to communicate with the passengers if such network exist. In order to connect to the *passenger intercom*, ACE3 Interaction Menu is needed: simply interact with the vehicle when being in a seat with access to *passenger intercom* and select connect. *Crew intercom* is automatically assigned to players in "crew" positions.
+While *crew intercom* is limited to crew members and players in cargo positions cannot have access to it, *pax intercom* can be accessed by both crew and pax. This allows the crew to communicate with the pax if such network exist. In order to connect to the *pax intercom*, ACE3 Interaction Menu is needed: simply interact with the vehicle when being in a seat with access to *pax intercom* and select connect. *Crew intercom* is automatically assigned to players in "crew" positions.
```
Comment (TheMagnetar) [inline docs/wiki/user/vehicle-intercom.md]:
```suggestion
While *crew intercom* is limited to crew members and players in cargo positions cannot have access to it, *pax intercom* can be accessed by both crew and passengers. This allows the crew to communicate with the passengers if such network exist. In order to connect to the *pax intercom*, ACE3 Interaction Menu is needed: simply interact with the vehicle when being in a seat with access to *pax intercom* and select connect. *Crew intercom* is automatically assigned to players in "crew" positions.
```

File: docs/wiki/user/vehicle-intercom.md [hunk 3]
```diff
@@ -50,10 +50,10 @@ It has 4 positions. When at PTT, the intercom will behave as *PTT activation* mo
 
 ## Infantry Telephone
 
-Infantry also has the possibility, in vehicles like tanks or IFVs, to communicate with the crew and/or passengers without entering the vehicle. To do so, face the vehicle, interact with it and take the infantry telephone by selecting the appropriate network if it is not in use already. You can either put it back, give it to another player or switch networks.
+Infantry also has the possibility, in vehicles like tanks or IFVs, to communicate with the crew and/or pax without entering the vehicle. To do so, face the vehicle, interact with it and take the infantry telephone by selecting the appropriate network if it is not in use already. You can either put it back, give it to another player or switch networks.
```
Comment (TheMagnetar) [inline docs/wiki/user/vehicle-intercom.md]:
```suggestion
Infantry also has the possibility, in vehicles like tanks or IFVs, to communicate with the crew and/or passengers without entering the vehicle. To do so, face the vehicle, interact with it and take the infantry telephone by selecting the appropriate network if it is not in use already. You can either put it back, give it to another player or switch networks.
```

File: docs/wiki/user/vehicle-intercom.md [hunk 4]
```diff
@@ -50,10 +50,10 @@ It has 4 positions. When at PTT, the intercom will behave as *PTT activation* mo
...
 
 Telephone can be mounted on the hull of the vehicle or in the center of the vehicle. If mounted on the hull (requires config support), maximum distance is 1.5m, otherwise it is 10m. The telephone will automatically be returned if the distance exceeds the maximum, either by player movement or vehicle movement.
 
-Crew members (not passengers) can make the infantry telephone ring as long as it is not in use. In order to do so, interact with the vehicle from one of the allowed positions (driver, commander, gunner and turret positions) and select start/stop ringing. The ringing will automatically stop once the infantry telephone is picked up, the vehicle is destroyed or no crew members are aboard.
+Crew members (not pax) can make the infantry telephone ring as long as it is not in use. In order to do so, interact with the vehicle from one of the allowed positions (driver, commander, gunner and turret positions) and select start/stop ringing. The ringing will automatically stop once the infantry telephone is picked up, the vehicle is destroyed or no crew members are aboard.
```
Comment (TheMagnetar) [inline docs/wiki/user/vehicle-intercom.md]:
```suggestion
Crew members (not passengers) can make the infantry telephone ring as long as it is not in use. In order to do so, interact with the vehicle from one of the allowed positions (driver, commander, gunner and turret positions) and select start/stop ringing. The ringing will automatically stop once the infantry telephone is picked up, the vehicle is destroyed or no crew members are aboard.
```

**Review by TheMagnetar — CHANGES_REQUESTED** [review]
Too much mass replace :D

**Comment by Vdauphin** [comment]
Perfect now :)

**Review by jonpas — APPROVED** [review]

**Review by TheMagnetar — APPROVED** [review]

**Pull request merged**